lib/common.sh

```
@@ -222,49 +455,26 @@ run_parallel() {
-		(
-			eval "$cmd"
-		) &
+		[ -z "$cmd" ] && continue
+		(eval "$cmd") &
+		pids+=($!)
 		running=$((running + 1))
 
 		if [ "$running" -ge "$max_jobs" ]; then
-			wait -n
+			# Wait for any job to complete (Bash 3.2 compatible)
+			wait ${pids[0]}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)